server/app/views/admin/programs/ProgramAdministratorProgramListView.java

```diff
@@ -162,7 +165,8 @@ private String extractProgramStatusText(
     throw new IllegalArgumentException("Program neither active nor draft.");
   }
 
-  Tag maybeRenderViewApplicationsLink(String text, Optional<ProgramDefinition> activeProgram) {
+  ContainerTag<?> maybeRenderViewApplicationsLink(
```

Comment:
I was mostly curious as to whether returning `Tag` rather than `ContainerTag` would work. This one's really minor, so if you'd prefer to keep as `ContainerTag` and it doesn't require any downstream wrapping, works for me.

Reply:
Ok, I changed a few to `Tag` and it compiled. Would you be able to run my CI tests before you leave? Since I'm a community contributor I need one of you all to run my tests